python/graphstorm/model/gnn.py

```diff
@@ -40,34 +40,56 @@ class GSOptimizer():
 
     Parameters
     ----------
-    dense_opts : list
-        A list of optimizer objects for dense model parameters.
+    dense_opts : th.optim.Optimizer, list or dict
```

Comment:
i don't understand. why do we want to accept a dict and explicitly tell the dense optimizer is for LM or for other models? I don't see how the optimizer wrapper handles them differently.

Reply:
Because we need to use different learning rate for dense model and language model.